### PR TITLE
Move model

### PR DESCRIPTION
**Pull request by BykhovDenis**
**Pull Request Requirements:**

- Provide a brief description of the changeset.
- Include a screenshots if applicable
- Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

### CODE REVIEW

File: models/notification/src/migration.ts [hunk 1]
```diff
@@ -266,7 +274,7 @@ export const notificationOperation: MigrateOperation = {
         state: 'remove-old-classes',
         func: async (client) => {
           await client.deleteMany(DOMAIN_NOTIFICATION, { _class: 'notification:class:DocUpdates' as Ref<Class<Doc>> })
-          await client.deleteMany(DOMAIN_TX, { objectClass: 'notification:class:DocUpdates' as Ref<Class<Doc>> })
+          await client.deleteMany(DOMAIN_MODEL_TX, { objectClass: 'notification:class:DocUpdates' as Ref<Class<Doc>> })
```
Comment (haiodo) [inline models/notification/src/migration.ts]: why?

File: models/notification/src/migration.ts [hunk 2]
```diff
@@ -402,14 +410,14 @@ export const notificationOperation: MigrateOperation = {
       {
         state: 'remove-update-txes-docnotify-ctx-v2',
         func: async (client) => {
-          await client.deleteMany(DOMAIN_TX, {
```
Comment (haiodo) [inline models/notification/src/migration.ts]: why?

File: models/notification/src/migration.ts [hunk 3]
```diff
@@ -402,14 +410,14 @@ export const notificationOperation: MigrateOperation = {
...
+          await client.deleteMany(DOMAIN_MODEL_TX, {
             _class: core.class.TxUpdateDoc,
             objectClass: notification.class.DocNotifyContext,
             'operations.lastViewedTimestamp': {
               $exists: true
             }
           })
-          await client.deleteMany(DOMAIN_TX, {
+          await client.deleteMany(DOMAIN_MODEL_TX, {
```
Comment (haiodo) [inline models/notification/src/migration.ts]: why?

File: tests/prepare.sh [hunk 4]
```diff
@@ -25,10 +25,6 @@ fi
 ./tool.sh create-account user2 -f Kainin -l Dirak -p 1234
 ./tool.sh create-account super -f Super -l User -p 1234
 ./tool.sh set-user-admin super true
-./tool.sh assign-workspace user1 sanity-ws
```
Comment (haiodo) [inline tests/prepare.sh:28]: why?

File: models/notification/src/migration.ts (hunk 3 above)
Comment (BykhovDenis) [reply models/notification/src/migration.ts]: Fixed

File: models/notification/src/migration.ts (hunk 2 above)
Comment (BykhovDenis) [reply models/notification/src/migration.ts]: Fixed

File: models/notification/src/migration.ts (hunk 1 above)
Comment (BykhovDenis) [reply models/notification/src/migration.ts]: Fixed

File: tests/prepare.sh (hunk 4 above)
Comment (BykhovDenis) [reply tests/prepare.sh:28]: We do it in restore

**Review by haiodo — APPROVED** [review]

**Pull request merged**